### PR TITLE
Abraham development

### PR DESCRIPTION
**Pull request by abrahamharos**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -35,6 +35,76 @@ <h2>About me.</h2>
...
+              <li>Matlab</li>
+              <li>Numerical Methods</li>
+            </ul>
+          </li>
+          <li>Backend for school exchange programs.
+            <ul>
+              <li>NodeJS</li>
+              <li>LoopBack Framework</li>
+            </ul>
+          </li>
```
Comment (akihiroota87) [inline portfolio/src/main/webapp/index.html]: This seems to have been listed twice. Did you intend to add a duplicate?

**Review by akihiroota87 — APPROVED** [review]
I had one comment, but otherwise looks good. Thanks for sharing more about yourself.

**Review by mschillaci — APPROVED** [review]
Looks good to me!

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (abrahamharos) [reply portfolio/src/main/webapp/index.html]: Oh, I forgot to change it. edited, thanks.

**Pull request merged**